front_end/entrypoints/rn_fusebox/FuseboxProfilingBuildObserver.ts

```diff
@@ -43,13 +48,17 @@ export default class FuseboxProfilingBuildObserver implements
 
   #handleMetadataUpdated(
       event: Common.EventTarget.EventTargetEvent<Protocol.ReactNativeApplication.MetadataUpdatedEvent>): void {
-    const {unstable_isProfilingBuild} = event.data;
+    const {unstable_isProfilingBuild, unstable_networkInspectionEnabled} = event.data;
```

Comment:
PR looks awesome. Can I just confirm that across this and https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/147:

- The `ENABLE_NETWORK_PANEL` experiment and tab treatment are ***separate*** from the existing Expo query string?

Then, let's rename this class: `FuseboxFeatureObserver`?

Reply:
Renamed, good catch! Updated comments too.

The experiment and the auto-enrollment is new and independent from the Expo query string. The Network panel loading/view extension registration will now be triggered on the Expo query string (old) _or_ this experiment (new).

For the `preview`/experimental tab treatment, I think it makes sense for both Expo and this Network panel to have it right? (I'd assume Expo's current offer is already showing it)

Reply:
> preview/experimental tab treatment

Yes makes sense for both, but I think we still want to differentiate the Expo version as before "Network (Expo, unstable) 🧪"

Reply:
Looks like https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/120 didn't make it to the `main` branch.

Added conditional title in this PR.

<img width="358" alt="image" src="https://github.com/user-attachments/assets/81962cc8-eb4a-4dd7-bb51-2ecee347888b" />